frontend/packages/pipelines-plugin/src/components/taskruns/list-page/TaskRunsListPage.tsx

```diff
@@ -29,6 +38,48 @@ const TaskRunsListPage: React.FC<Omit<
     }),
     [showPipelineColumn],
   );
+  const columnManagementID = referenceForModel(TaskRunModel);
+  const [tableColumns, ,] = useUserSettingsCompatibility<TableColumnsType>(
+    COLUMN_MANAGEMENT_CONFIGMAP_KEY,
+    COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY,
+    undefined,
+    true,
+  );
+  const getColumns = (): TableColumn<TaskRunKind>[] => [
+    {
+      title: t('public~Name'),
+      id: 'name',
+    },
+    {
+      title: t('public~Namespace'),
+      id: 'namespace',
+    },
+    {
+      title: t('public~Pipeline'),
+      id: 'pipeline',
+    },
+    {
+      title: t('public~Task'),
+      id: 'task',
+    },
+    {
+      title: t('public~Pod'),
+      id: 'pod',
+    },
+    {
+      title: t('public~Status'),
+      id: 'status',
+    },
+    {
+      title: t('public~Started'),
+      id: 'started',
+    },
+    {
+      title: t('public~Duration'),
+      id: 'duration',
+      additional: true,
```

Comment:
Translation namespace should be `pipelines-plugin`

```suggestion
  const getColumns = (): TableColumn<TaskRunKind>[] => [
    {
      title: t('pipelines-plugin~Name'),
      id: 'name',
    },
    {
      title: t('pipelines-plugin~Namespace'),
      id: 'namespace',
    },
    {
      title: t('pipelines-plugin~Pipeline'),
      id: 'pipeline',
    },
    {
      title: t('pipelines-plugin~Task'),
      id: 'task',
    },
    {
      title: t('pipelines-plugin~Pod'),
      id: 'pod',
    },
    {
      title: t('pipelines-plugin~Status'),
      id: 'status',
    },
    {
      title: t('pipelines-plugin~Started'),
      id: 'started',
    },
    {
      title: t('pipelines-plugin~Duration'),
      id: 'duration',
      additional: true,
```